### PR TITLE
Addling links to promote internships

### PR DESCRIPTION
**Pull request by philspray**
### Trello card

https://trello.com/c/zRPuUndY

### Context

Once the internship programme is open, we need to promote it around the site.

### Changes proposed in this pull request

Adding a section to the 'Decide who to teach' step on the steps to become a teacher.

### CODE REVIEW

File: app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb [hunk 1]
```diff
@@ -8,3 +8,5 @@
 </ul>
 
 <p>Follow these steps if you want to teach primary or secondary, including <a href="/teach-disabled-pupils-and-pupils-with-special-educational-needs">teaching disabled pupils and pupils with special educational needs</a> in these settings.<p>
+
+<p>If you’re an undergraduate, interested in teaching chemistry, computing, languages, maths or physics then a <a href="/teaching-internship-providers">teaching internship</a> might be for you. It lasts for 3 weeks, starts in June and you’ll be paid £300 per week while you experience the classroom.</p>
```
Comment (EllieNodder) [inline app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb:12]:
```suggestion
<p>If you’re an undergraduate interested in teaching chemistry, computing, languages, maths or physics then you could do a <a href="/teaching-internship-providers">paid teaching internship</a> to find out if teaching is right for you.</p>
```
I just wonder if the 'lasts for 3 weeks, starts in June ... etc.' is a bit too much information for this point? Also makes that para a bit shorter/easier to read. Feel free to completely disagree!

**Comment by gemmadallmandfe** [comment]
@philspray I had a look at what we did last year: https://github.com/DFE-Digital/get-into-teaching-app/pull/2250

* The `/train-to-be-a-teacher/get-school-experience` page definitely needs updating - it doesn't cover all the subjects for a start - does it need anything else adding?
* The `/subjects/maths` page is not inaccurate - does it need beefing up at all?
* The blog pages that were updated last year hardly drove any traffic to the internships page
* Is there anywhere else where it would make sense to have a link (may not be)?

I've also looked at previous page paths in GA from last year - I'll add a link to the Trello ticket

**Comment by gemmadallmandfe** [comment]
@philspray Also - there's a linting error so the PR hasn't built successfully :-(

File: app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb [hunk 2]
```diff
@@ -7,4 +7,6 @@
   <li><a href="/become-a-further-education-teacher">further education</a> (for example, teaching at further education colleges or sixth form colleges)</li>
 </ul>
 
-<p>Follow these steps if you want to teach primary or secondary, including <a href="/teach-disabled-pupils-and-pupils-with-special-educational-needs">teaching disabled pupils and pupils with special educational needs</a> in these settings.<p>
+<p>Follow these steps if you want to teach primary or secondary, including <a href="/teach-disabled-pupils-and-pupils-with-special-educational-needs">teaching disabled pupils and pupils with special educational needs</a> in these settings.</p>
+
+<p>If you’re an undergraduate, interested in teaching chemistry, computing, languages, maths or physics then you could do a <a href="/teaching-internship-providers">paid teaching internship</a> to find out if teaching is right for you.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb:12]:
```suggestion
<p>If you’re an undergraduate and interested in teaching chemistry, computing, languages, maths or physics then you could do a <a href="/teaching-internship-providers">paid teaching internship</a> to find out if teaching is right for you.</p>
```

not sure about the first comma in terms of readability @philspray? Is 'and' easier to understand?

Comment (philspray) [reply app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb:12]: Yep I agree, and is better

File: app/views/content/subjects/maths/_article.html.erb [hunk 3]
```diff
@@ -149,11 +149,10 @@
...
     <h2 class="heading-s">Maths internships</h2>
 
-    <p>
-    If you’re currently doing an undergraduate degree and are interested in
-    teaching maths, you could apply for a paid <a href="/teaching-internship-providers">teaching internship</a>. This gives
-    you 3 weeks paid experience to show you what it’s like to be in a classroom.
-    </p>
+    <p>If you’re currently doing an undergraduate degree and are interested in teaching maths, you could apply for a paid teaching internship. </p>
+    <p> It could help you to understand what it’s really like in the classroom and get a feel for school life.</p>
+     <p>Internships last for 3 weeks, starts in June and you’ll be paid £300 per week while you experience the classroom.</p>
```
Comment (EllieNodder) [inline app/views/content/subjects/maths/_article.html.erb:154]:
```suggestion
     <p>Internships last for 3 weeks, start in June and you’ll be paid £300 per week while you experience the classroom.</p>
```
I think? just read a bit weird to me

**Review by EllieNodder — APPROVED** [review]
Looks good to me!

**Pull request merged**